entry.go

```diff
@@ -208,13 +181,67 @@ func (te *TunaEntry) StartReverse(stream *smux.Stream) error {
 			return
 		}
 		stream.Close()
+
+		lastComputed := common.Fixed64(0)
```

Comment:
L185-L233 is a large chunk of duplicate code in exit `handleSession` function. You need to find a way to combine theses codes together.